README.md

```diff
@@ -1,5 +1,5 @@
 # TICF Zendesk integration with TxMA
-
+TESTNIG
```

Comment:
Is this a typo?

Reply:
You guys are too quick! This was just a test PR to see what's going on with the build!